src/synthgauge/evaluate.py

```diff
@@ -16,7 +18,24 @@
 
 
 class Evaluator(object):
-    """ """
+    """
+    The central class in ``synthgauge`` used to hold and evaluate data via
+    metrics or visualisation.
+
+    Parameters
+    ----------
+    real: pandas.DataFrame
+        The real, observed dataset to be evaluated against.
```

Comment:
```suggestion
        Dataframe containing the real data.
```
Updated for consistency with similar docstrings in the metrics modules. I appreciate this is now less descriptive but should still be clear from context what `real` (and `synth`) data is.